dashboards-observability/query_manager/ast/types/stats.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * stats chunck types
+ */
+export interface StatsAggregationChunk {
+  function: StatsAggregationFunctionChunk;
+  function_alias: string;
+}
```

Comment:
We can move this file to constants as well.

Reply:
Please refer to above comments.